### PR TITLE
Feature iops 2581 add bars code system message event bars to nhse ig

### PR DESCRIPTION
**Pull request by SheenaSharma**

### CODE REVIEW

**Review by ryma2fhir — APPROVED** [review]

File: CodeSystem-England-MessageEventsBars.xml
Comment (digitalhealthmartin) [inline CodeSystem-England-MessageEventsBars.xml:1]: This file needs moving to codesystems folder

Comment (digitalhealthmartin) [reply CodeSystem-England-MessageEventsBars.xml:1]: BARS may be better than Bars

File: CodeSystem-England-MessageEventsBars.xml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  <contact>
+    <name value="NHS England" />
+    <telecom>
+      <system value="email" />
+      <value value="interoperabilityteam@nhs.net" />
+      <use value="work" />
+      <rank value="1" />
+    </telecom>
+  </contact>
+    <description value="Common messaging events for Bars Service Requests." />
```
Comment (digitalhealthmartin) [inline CodeSystem-England-MessageEventsBars.xml]: Suggest: A set of codes that define common messaging events for BARS Service Requests.

File: codesystems/CodeSystem-England-MessageReasonBars.xml [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+  <contact>
+    <name value="NHS England" />
+    <telecom>
+      <system value="email" />
+      <value value="interoperabilityteam@nhs.net" />
+      <use value="work" />
+      <rank value="1" />
+    </telecom>
+  </contact>
+    <description value="Message Reason codes." />
```
Comment (digitalhealthmartin) [inline codesystems/CodeSystem-England-MessageReasonBars.xml]: Suggest: A set of codes that define common messaging reasons for BARS Service Requests.

File: codesystems/CodeSystem-England-MessageReasonBars.xml
Comment (digitalhealthmartin) [inline codesystems/CodeSystem-England-MessageReasonBars.xml:1]: Similar to other file, maybe make Bars to BARS.

**Review by digitalhealthmartin — CHANGES_REQUESTED** [review]

File: CodeSystem-England-MessageEventsBars.xml
Comment (SheenaSharma) [reply CodeSystem-England-MessageEventsBars.xml:1]: Changes are done.

**Review by SheenaSharma — COMMENTED** [review]
Changes are done. Please review it.

**Review by digitalhealthmartin — APPROVED** [review]

**Pull request merged**